snap-extras/sitecustomize.py

```diff
@@ -0,0 +1,8 @@
+import site
+import os
+
+snap_dir = os.getenv("SNAP")
+if snap_dir:
```

Comment:
This might make it a bit less error prone (running a terminal that is a SNAP like the VS Code shell) and setting the wrong thing up if this ever becomes more that snap-extras (which I suppose is why you added the conditional) :-)

```suggestion
if snap_dir and os.getenv("SNAP_NAME") == "charmcraft":
```